include/mbgl/renderer/renderer_state.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <mbgl/map/camera.hpp>
+#include <mbgl/util/geo.hpp>
+
+namespace mbgl {
+
+class UpdateParameters;
+
+class RendererState {
```

Comment:
Why is this a class? It seems to only have one static function member. Could it be a standalone function instead?

Reply:
This class acts as a container for renderer state-like functions - I plan to add a few other letters, but  I'm :+1: in converting this into e.g. a sub-namespace instead.